### ISSUE TITLE
preferential lanes - professional bus driver

### ISSUE THREAD

**Issue by Lore2688** [osmandapp/OsmAnd-iOS#3763]
Hi there,
i'm a professional bus driver (public local bus service inside city) and i have a gpx file that i use to move inside the city (yes, i'm new in this work so can be very helpful have these files in osmand). My problem is:  inside the city center we have dedicated preferential lanes but osmand recalculate the route because, for him, these are against traffic. Can be solved this problem? thank you

**Comment by yuriiurshuliak** [comment]
Could you share the coordinates of the bus lanes in question? Also, let us know which version of the OsmAnd app you're using. It would be helpful if you could provide a short video or some screenshots that illustrate the issue.

**Comment by Lore2688** [comment]
> Could you share the coordinates of the bus lanes in question? Also, let us know which version of the OsmAnd app you're using. It would be helpful if you could provide a short video or some screenshots that illustrate the issue.

For sure..this is the link to Via Erbosa (Florence italy) https://www.openstreetmap.org/#map=19/43.75948/11.29072 but this happen not only here...sometimes osmand have no problem with bus lanes, other times it change the route as you can see in the screenshot (i'll post google drive link under).
I'm using osmand 4.7.6 on iphone 15 pro max 

https://drive.google.com/file/d/1ygOAX_fKgnl0iFeIhPyNaA1nXewY-szI/view?usp=share_link
https://drive.google.com/file/d/110QgevxUqkoYlTtWGIl4njcBU4RQN6gO/view?usp=share_link
https://drive.google.com/file/d/1SmKF8Vav-POspdJL21eoO2lVhW0jrU1Z/view?usp=share_link
https://drive.google.com/file/d/1hNAN3x9DwDZY5F8tlVbeeQ7nfgTizQ34/view?usp=share_link

**Comment by yuriiurshuliak** [comment]
Do I understand correctly that you are asking for support for this tag? [lanes:psv](https://wiki.openstreetmap.org/wiki/Key:lanes:psv:backward?uselang=en)

Or are you also requesting a separate configuration or bus lanes function?

Also, please keep in mind that your screenshots are not visible, because after following these links, you need access to see them.

**Comment by Lore2688** [comment]
![IMG_7413](https://github.com/osmandapp/OsmAnd-iOS/assets/169367212/bb091be6-5833-4b15-b12f-ca5cb551d78d)

**Comment by Lore2688** [comment]
Yes, problem is about bus lane as you can see in this screen, osmand avoid these bus lane but i need to follow this track!
